universal-application-tool-0.0.1/app/services/question/QuestionDefinition.java

```diff
@@ -59,23 +57,20 @@ public String getDescription() {
     return this.description;
   }
 
-  /** Localized question text. Keys are ISO language codes, and values are the translated text. */
-  public ImmutableMap<Locale, String> getQuestionText() {
-    return this.questionText;
-  }
-
-  /**
-   * Localized question help text. Keys are ISO language codes, and values are the translated text.
-   */
-  public ImmutableMap<Locale, String> getQuestionHelpText() {
-    return this.questionHelpText;
+  /** Get the question text for the given locale. */
+  public String getQuestionText(Locale locale) {
+    if (this.questionText.containsKey(locale)) {
+      return this.questionText.get(locale);
+    }
+    throw new RuntimeException("Locale not found: " + locale);
```

Comment:
Suggestion (non-blocking): please add a space between the `if` and the `throw` 🙏🏻